test/custom-operations/anonymous-naming.spec.ts

```diff
@@ -0,0 +1,177 @@
+import { Parser } from '../../src/parser';
+import { xParserMessageName, xParserSchemaId } from '../../src/constants';
+
+describe('custom operations - anonymous naming', function() {
+  const parser = new Parser();
+
+  it('should assign x-parser-message-name to anonymous message', async function() {
+    const { document } = await parser.parse({
+      asyncapi: '2.0.0',
+      info: {
+        title: 'Valid AsyncApi document',
+        version: '1.0',
+      },
+      channels: {
+        channel: {
+          publish: {
+            operationId: 'operation',
+            message: {
+              payload: {}
+            }
+          }
+        }
+      }
+    });
+
+    expect(document?.messages()).toHaveLength(1);
```

Comment:
Isn't like this? I'm confused now 😅 
```suggestion
    expect(document?.messages().all()).toHaveLength(1);
```

Reply:
Collection class extends Array, so `messages()` returns exactly array of data (with additional methods).